### DISCUSSION TITLE
Odd choice of icons and no hover

### DISCUSSION THREAD

**Discussion by greglow-sdu** [tabulartools/pbi-explorer#10] [Q&A] [answered]
In the main UI, in the left-hand pane, the choice of icons at the top seems odd, and it's a pity that there doesn't seem to be a hover action that explains the purpose of the icon when you're above it, like there is for the icons at the bottom of the screen.

The legal terms icon looks like some sort of "new message" or "new report" or "edit" icon.
The bug icon (triangle) doesn't seem to bear any resemblance to a bug, unless it's meant to be part of a hazard warning or something. 
The lightbulb is probably ok, and the info icon is fine.

**Comment by DBojsen** [comment, marked as answer]
Hi @greglow-sdu 

Completely agree - we need to improve these.
And of course they should all have hover effect.

Until we fix, you can expand the menu by pressing the top botton, which will show the descriptions for each menu item.

BR David - Tabular Tools

**Comment by DBojsen** [comment]
Hi @greglow-sdu 

This will fixed in BETA3, coming out in a few days.
